app/js/views/main.js

```diff
@@ -104,6 +104,29 @@ var mainViewTemplate =
 
     --checkbox-border-color: var(--background-minus-minus);
   }
+
+  fxos-customizer {
+    background-color: rgba(0, 0, 0, 0.8);
+    position: fixed;
+    left: 0;
+    right: 0;
+    top: 100%; /* off-screen by default, animated translate to show and hide */
+    height: 50%;
+    border-top: 1px solid #ccc;
+    /*
+     * this needs to go on top of the regular app, but below
+     * gaia-modal and gaia-dialog which we override elsewhere.
```

Comment:
A bit nit-picky, but this is overly-specific. We may add more components or remove the ones here, and it's very likely that we'll forget about this comment.